packages/client/src/app/game/views/game-map/me/me.component.ts

```diff
@@ -69,6 +74,10 @@ export class MeComponent implements OnInit, OnDestroy {
       this.character.state$.map(meState => meState && meState.state === MeModelState.SHOOTING))
       .map((result => result[0] || result[1]));
     this.showCross$ = this.character.state$.map(meState => meState && meState.state === MeModelState.SHOOTING);
+    this.meModelDrawSubscription = this.meModel.onDraw.subscribe(entity => {
```

Comment:
use `getCesiumObjectsMap()` instead of onDraw

Reply:
we need to get the updated cesiumObject every draw

Reply:
dont we do object assign every update?